src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -59,7 +59,7 @@ public class EditCommand extends Command {
     private final EditPersonDescriptor editPersonDescriptor;
 
     /**
-     * @param index of the person in the filtered person list to edit
+     * @param index                of the person in the filtered person list to edit
```

Comment:
```suggestion
     * @param index of the person in the filtered person list to edit
```

Reply:
I think it was due to IntelliJ's formatting to try and keep the descriptions of the params in line. Good catch!